apps/aecore/src/aec_sync.erl

```diff
@@ -91,12 +91,12 @@ compare_ping_objects(Local, Remote) ->
     lager:debug("Compare, Local: ~p; Remote: ~p", [pp(Local), pp(Remote)]),
     Src = maps:get(<<"source">>, Remote),
     Res = case {maps:get(<<"genesis_hash">>, Local),
-                maps:get(<<"genesis_hash">>, Remote)} of
+                maps:get(<<"genesis_hash">>, Remote, missing)} of
               {G, G} ->
                   lager:debug("genesis blocks match", []),
                   %% same genesis block - continue
                   case {maps:get(<<"best_hash">>, Local),
-                        maps:get(<<"best_hash">>, Remote)} of
+                        maps:get(<<"best_hash">>, Remote, missing)} of
```

Comment:
This check shall be removed. Rather, the [`swagger.yaml` definition for `Ping`](https://github.com/aeternity/epoch/blob/bb5786751abdff67cc667092e7dd5e7ae45d6724/config/swagger.yaml#L360) shall be enriched with [`required`](https://github.com/OAI/OpenAPI-Specification/blob/1adcb8830aab48506b1ffd876330ab8c22820ad6/versions/2.0.md#schema-object).

Reply:
Will be addressed in [Pivotal story](https://www.pivotaltracker.com/story/show/153629617)